poglink/config.py

```diff
@@ -32,6 +33,8 @@
     "rates_urls",
 ]
 
+BOOLEAN_VALUES = ["publish_on_startup"]
```

Comment:
Only one right now, but this would require less refactoring later if we added another bool type.